test/unit/puppetlabs/puppetserver/certificate_authority_test.clj

```diff
@@ -2,7 +2,7 @@
   (:import (java.io StringReader ByteArrayInputStream ByteArrayOutputStream))
   (:require [puppetlabs.puppetserver.certificate-authority :refer :all]
             [puppetlabs.trapperkeeper.testutils.logging :as logutils]
-            [puppetlabs.certificate-authority.core :as utils]
+            [puppetlabs.ssl-utils.core :as utils]
```

Comment:
Ahh ok, that makes sense. And this was as fairly IntelliJ/Cursive-centric comment.